src/Components/tournament-rounds-form/index.tsx

```diff
@@ -41,27 +44,39 @@ export const TournamentRoundForm = (props: {id, startDate, endDate}) => {
             return;
         }
 
+        let newRound
+
+        if(props.number === -1){
+            newRound = {date: date, number: props.number, numberOfRuns: numberOfRuns, teamId: 1}
```

Comment:
ID need to be obtain from database probably, or we should refresh data in tournament-rounds